components/raftstore-v2/src/operation/command/admin/flashback.rs

```diff
@@ -7,7 +7,11 @@ use kvproto::{
     raft_serverpb::RegionLocalState,
 };
 use protobuf::Message;
-use raftstore::{coprocessor::RegionChangeReason, store::metrics::PEER_ADMIN_CMD_COUNTER, Result};
+use raftstore::{
```

Comment:
please also track the # of kv scanned to locate the keys to flashback and # of kv scanned to find the proper version.